integration_tests/build_test_containers.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$(docker images -q ${D3A_IMAGE_TAG} 2> /dev/null)" == "" ]]; then
   echo "Building d3a image ..." && \
   rm -rf tests/d3a && \
   cd tests/ && \
-  git clone https://github.com/gridsingularity/d3a.git && \
+  git clone -b feature/D3ASIM-3333 https://github.com/gridsingularity/d3a.git && \
```

Comment:
We can restore the original version of this and point to `master` now, right?

Reply:
yes, already created a local commit. Thanks!